doc/manual/plugins_local/crosshair.rst

```diff
@@ -4,7 +4,7 @@ Crosshair
 =========
 
 .. image:: figures/crosshair-plugin.png
```

Comment:
I think this screenshot is outdated too, especially now you have moved it to the left and the UI has changed completely. I think we need to update this.

Reply:
Argh, I am sorry. I was looking at Cuts, the wrong plugin to review! It's Monday morning. Please ignore this.